openapi.yml

```diff
@@ -2806,6 +5502,140 @@ components:
           description: The total amount of memory available in bytes
           minimum: 1
 
+    MultipartUpload:
```

Comment:
This seems to be unused. Same for `MultipartListPartItem`, `MultipartCompletedPart` and `Sector`